strax/corrections.py

```diff
@@ -0,0 +1,120 @@
+'''I/O format for corrections using MongoDB
+
+This module list, reads, writes, among others functionalities
+to a MongoDB where corrections are stored.
+'''
+
+from datetime import datetime
+import logging
+import pandas as pd
+import pdmongo as pdm
+import pymongo
+
+import strax
+
+export, __all__ = strax.exporter()
+
+
+@export
+class CorrectionsInterface:
+    '''A class to manage corrections that are stored in a MongoDB.
+    '''
+
+    def __init__(self, host='127.0 0.1', username=None, password=None,
+                 database_name='corrections'):
+
+        self.host = host
+        self.username = username
+        self.password = password
+        self.client = pymongo.MongoClient(host=self.host,
+                                          username=self.username,
+                                          password=self.password)
+        self.database_name = database_name
+        self.database = self.client[self.database_name]
+
+    def list_corrections(self):
+        '''Smart logic to list corrections.
+        '''
+        return [x['name'] for x in self.database.list_collections()
+                if 'global' not in x['name']]
+
+    def read(self, correction):
+        '''Smart logic to read corrections,
+        where correction is the name of the DataFrame.
+        '''
+        df = pdm.read_mongo(correction, [], self.database)
+
+        # No data found
+        if df.size == 0:
+            return None
+        # Delete internal Mongo identifier
+        del df['_id']
+        return df.set_index('time')
+
+    def interpolate(self, what, when, how='interpolate'):
+        '''Interpolate values of a given correction(what=DataFrame)
```

Comment:
This is a comment for all your docstrings. Could you please check the sphinx syntax https://pythonhosted.org/an_example_pypi_project/sphinx.html and adopt you doc-strings accordingly. I know that for most of the functions in strax this was not done. But I would love if we could start doing this to have a nice documentation. See as an example in strax: https://github.com/AxFoundation/strax/blob/7e5ee60bf1c094f296beece469520840e2fbadef/strax/chunk.py#L121

Reply:
Further you should maybe list all the possible `how`option.

Reply:
Thanks for pointing this, this will benefit every user including me. Done